xla/service/gpu/runtime/command_buffer_cmd.h

```diff
@@ -203,8 +203,12 @@ class CommandBufferCmd {
   // Returns a command execution scope computed from the command stream id and
   // the default command buffer execution scope.
   se::CommandBuffer::ExecutionScopeId GetExecutionScope(
-      const RecordParams& record_params) const;
+      const CommandBufferCmd::RecordParams& record_params,
```

Comment:
Make it static? Otherwise hard to distinguish from the next one

Reply:
Don't quite understand, why making it static will distinguish from the next GetExecutionScope?